chart/templates/scheduler/scheduler-deployment.yaml

```diff
@@ -53,7 +53,7 @@ metadata:
     release: {{ .Release.Name }}
     chart: "{{ .Chart.Name }}-{{ .Chart.Version }}"
     heritage: {{ .Release.Service }}
-    executor: {{ .Values.executor }}
+    executor: {{ trunc -50 .Values.executor }}  # AWS ECS Executor name is too long.
```

Comment:
I was looking into the kubernetes docs and I'm reading that the limit is 63 characters, is there any reason we're limiting to 50 here?

source: https://kubernetes.io/docs/concepts/overview/working-with-objects/labels/

Reply:
You are right @o-nikolas . I double checked, it's 63 characters max. I will change it.

Reply:
It's now 63.